src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/components/__tests__/MethodologyStatusForm.test.tsx

```diff
@@ -0,0 +1,256 @@
+          } as BasicMethodology
+        }
+        showWithRelease // EES-2163 flag
+        onCancel={noop}
+        onSubmit={noop}
+      />,
+    );
+
+    expect(screen.getByText('Edit methodology status')).toBeInTheDocument();
+    expect(screen.getByText('Status')).toBeInTheDocument();
+
+    const statuses = within(
+      screen.getByRole('group', { name: 'Status' }),
+    ).getAllByRole('radio');
+
+    expect(statuses).toHaveLength(2);
+    expect(statuses[0]).toHaveAttribute('value', 'Draft');
+    expect(statuses[0]).toBeEnabled();
+    expect(statuses[0]).toBeChecked();
+    expect(screen.getByLabelText('In draft')).toBeInTheDocument();
+
+    expect(statuses[1]).toHaveAttribute('value', 'Approved');
+    expect(statuses[1]).toBeEnabled();
+    expect(
+      screen.getByLabelText('Approved for publication'),
+    ).toBeInTheDocument();
+
+    expect(
+      screen.queryByRole('group', { name: 'When to publish' }),
+    ).not.toBeInTheDocument();
+  });
+
+  test('renders the form with approved status initial values', () => {
+    render(
+      <MethodologyStatusForm
+        methodologySummary={
+          {
+            status: 'Approved',
+            latestInternalReleaseNote: 'The latest note',
+          } as BasicMethodology
+        }
+        showWithRelease // EES-2163 flag
+        onCancel={noop}
+        onSubmit={noop}
+      />,
+    );
+
+    const approvedRadio = screen.getByLabelText('Approved for publication');
+    expect(approvedRadio).toBeChecked();
+
+    const noteField = screen.getByLabelText('Internal note');
+    expect(noteField).toHaveValue('The latest note');
+
+    expect(
+      screen.queryByRole('group', { name: 'When to publish' }),
+    ).toBeInTheDocument();
+  });
+
+  test('renders the form with publishing strategy initial values', () => {
+    render(
+      <MethodologyStatusForm
+        methodologySummary={
+          {
+            status: 'Approved',
+            latestInternalReleaseNote: 'The latest note',
+            publishingStrategy: 'WithRelease',
+            withReleaseId: 'test-release-2',
+          } as BasicMethodology
+        }
+        showWithRelease // EES-2163 flag
+        unPublishedReleases={testUnpublishedReleases}
+        onCancel={noop}
+        onSubmit={noop}
+      />,
+    );
+
+    expect(screen.getByText('When to publish')).toBeInTheDocument();
+
+    const publishStatuses = within(
+      screen.getByRole('group', { name: 'When to publish' }),
+    ).getAllByRole('radio');
+
+    expect(publishStatuses[0]).toBeChecked();
+    expect(publishStatuses[0]).toBeEnabled();
+    expect(
+      screen.getByLabelText('With a specific release'),
+    ).toBeInTheDocument();
+
+    expect(publishStatuses[1]).not.toBeChecked();
+    expect(publishStatuses[1]).toBeEnabled();
+    expect(screen.getByLabelText('Immediately')).toBeInTheDocument();
+
+    const releaseSelect = screen.getByLabelText('Select release');
+    expect(releaseSelect.children.length).toBe(3);
```

Comment:
Could check that all the options look correct using `toHaveTextContent` and `toHaveValue`?